Unclear getting started instructions for contributors

**Is your feature request related to a problem? Please describe.**

I'm trying to follow-up on https://github.com/dotansimha/graphql-yoga/issues/2655, and managed to consistently reproduce this and better identify how to write a failing test for the problem. However, I don't know how to get started after cloning this repository. The general [Guild contributing guide](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md) linked to as the only guidance for contributors to this repo doesn't have concrete steps to start. The closest thing there is that it mentions yarn, but having tried running with yarn this hasn't worked.

I _think_ based on inspecting a few of the package.json scripts I need to set up pnpm and run some of its commands. However, trying a naive `npm i -g pnpm && pnpm install` seems to result in lots of warnings about peer dependencies, and `pnpm test` or `npm test` does not then work (gives error `Cannot find module '../graphiql-html.js' from 'packages/graphql-yoga/src/plugins/useGraphiQL.ts'`).

In any case, I think it's definitely unclear what I need to do here - and really I shouldn't need to decipher it from the package.json: there should be a few commands suggested as a starting point for new contributors.

**Describe the solution you'd like**

A clear and concise guide for new contributors to quickly be able to follow the steps in the general Guild contributing guide. For example:
1. What dependencies do I need to install to build the project and run the tests?
2. Where do I probably want to start if I want to add a test

**Describe alternatives you've considered**

Standardising the way contributing to this project works with other the-guild projects and documenting it clearly centrally.

**Additional info**

My current understanding of the actual correct process for contributors looks like (but this is from trial+error and guesswork):

1. Install [Node.js](https://nodejs.org/)
2. Run in your terminal: `npm i -g pnpm@7 && pnpm install && pnpm build`
3. Add tests to `packages/graphql-yoga/__tests__` using [Jest](https://jestjs.io/docs/api) APIs
4. Run the tests with `pnpm test`

--- a/packages/graphql-yoga/README.md
+++ b/packages/graphql-yoga/README.md
@@ -99,6 +99,13 @@ Read more about how GraphQL Yoga compares to other servers in the ecosystem [her
 
 If this is your first time contributing to this project, please do read our [Contributor Workflow Guide](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md) before you get started off.
 
+For this project in particular, to get started on `stage/2-failing-test`:
+
+1. Install [Node.js](https://nodejs.org/)
+2. Run in your terminal: `npm i -g pnpm@7 && pnpm install && pnpm build`
+3. Add tests to `packages/graphql-yoga/__tests__` using [Jest](https://jestjs.io/docs/api) APIs
+4. Run the tests with `pnpm test`
+
 Feel free to open issues and pull requests. We're always welcome support from the community.
 
 ## Code of Conduct

